Sources/ProjectDescription/ProjectOption.swift

```diff
@@ -2,13 +2,28 @@ import Foundation
 
 /// Additional options related to the `Project`
 public enum ProjectOption: Codable, Equatable {
+    /// Defines how to group targets into scheme for autogenerated schemes
+    public enum AutomaticSchemesGrouping: Codable, Equatable {
+        /// Group all the targets of the project in a single scheme
+        case singleScheme
```

Comment:
Is this to create just the `-Project` scheme?

Reply:
Actually I forgot we already have that, the idea was to have just one scheme with everything inside. So I guess in this case we can go with the `-Project` one 🚀

Reply:
Actually this is different.
This scheme would be per Project, while the other would be per Workspace, right?